mava/components/building/distributor.py

```diff
@@ -37,6 +37,8 @@ class DistributorConfig:
     single_process_max_episodes: Optional[int] = None
     is_test: Optional[bool] = False
     wait: Optional[bool] = False
+    metrics_checkpoint: Tuple = ("mean_episode_return",)
```

Comment:
I would say maybe add a comment here explaining what this variable does, as it is not obvious. Something like `# saves networks based on this metric`. 

For the same reason I'd rename the variable to `checkpointing_metric`, but this is debatable, you can leave it as is if you feel that is better.